environment.yml

```diff
@@ -16,6 +16,8 @@ dependencies:
   - click
   - openeye-toolkits
   - typing-extensions
+  - pydantic
+  - openff-units
   - pip:
     - git+https://github.com/OpenFreeEnergy/Lomap@main
```

Comment:
Not in scope of PR but, lomap is on conda-forge now so we can move this, unless we need to use the @main version, and if we do, we should get that onto conda-forge.

Reply:
So I sort of agree here, but it also wouldn't hurt to use the "develop version" of lomap for now. That way if we pick up issues in lomap we can quickly fix, check it works and release a new lomap version rather than having to wait for conda-forge to update and then realising that it's not fixed etc...

Reply:
Not a bad idea, I've forgotten to fix this twice now :sweat_smile: so lets just keep